acceptance_tests/features/steps/sensitive_data.py

```diff
@@ -0,0 +1,43 @@
+import json
+from time import sleep
+
+from behave import step
+
+from acceptance_tests.utilities.database_helper import open_write_cursor
+from acceptance_tests.utilities.rabbit_helper import publish_json_message
+from acceptance_tests.utilities.test_case_helper import test_helper
+from config import Config
+
+
+@step("an UPDATE_SAMPLE_SENSITIVE event is received")
+def send_update_sample_sensitive_msg(context):
+    message = json.dumps(
+        {
+            "event": {
+                "type": "UPDATE_SAMPLE_SENSITIVE",
+                "source": "RH",
+                "channel": "RH",
+                "dateTime": "2021-06-09T13:49:19.716761Z",
+                "transactionId": "92df974c-f03e-4519-8d55-05e9c0ecea43"
+            },
+            "payload": {
+                "updateSampleSensitive": {
+                    "caseId": context.emitted_cases[0]['caseId'],
+                    "sampleSensitive": {"PHONE_NUMBER": "07898787878"}
+                }
+            }
+        })
+
+    publish_json_message(message, exchange=Config.RABBITMQ_EVENT_EXCHANGE,
+                         routing_key=Config.RABBITMQ_UPDATE_SAMPLE_SENSITIVE_QUEUE)
+
+
+@step("the sensitive data on the case is changed")
+def sensitive_data_on_case_changed(context):
+    sleep(3)
```

Comment:
Best to avoid hardcoded sleeps, instead it should retry with a timeout.

Check out this helper function we had census-rm-acceptance tests https://github.com/ONSdigital/census-rm-acceptance-tests/blob/872198234e13c421c8a6acc16175a48f435b6dcf/acceptance_tests/utilities/database_helper.py#L38